app/org/zalando/hutmann/authentication/OAuth2Action.scala

```diff
@@ -31,8 +30,9 @@ class OAuth2Action(
   val filter:         User => Future[Boolean] = { user: User => Future.successful(true) },
   val autoReject:     Boolean                 = true,
   val requestTimeout: Duration                = 1.seconds
-)(implicit config: Config, ec: ExecutionContext, ws: WSClient, materializer: Materializer)
-    extends ActionBuilder[UserRequest] {
+)(implicit config: Config, val executionContext: ExecutionContext, ws: WSClient, materializer: Materializer,
```

Comment:
I cannot comment on the actual line. Play 2.6 deprecates the `Action` object which is used on l. 178. Do you reckon it's possible to change this into something that isn't deprecated? I guess one could accept an implicit `ActionBuilder` but I'm not sure what this will do in practice.